exercises/practice/pangram/.docs/introduction.md

```diff
@@ -0,0 +1,16 @@
+# Introduction
+
+You work for a company that sells fonts through their website.
+They'd like to show a different sentence each time someone views a font on their website.
+To give a comprehensive sense of the font, the random sentences should use **all** the letters in the English alphabet.
```

Comment:
Especially, continuing from feedback above, we mention "all the letters" it could be helpful to list what those letters are.